psi4/src/psi4/occ/cepa_iterations.cc

```diff
@@ -55,32 +55,35 @@ void OCCWave::cepa_iterations() {
                  // DIIS
     DIISManager t2_diis;
     if (nooA + nooB != 1) {
-        psio_->open(PSIF_OCC_DPD, PSIO_OPEN_OLD);
-        t2_diis = DIISManager(maxdiis_, "CEPA DIIS T2 Amps", DIISManager::RemovalPolicy::LargestError, DIISManager::StoragePolicy::OnDisk);
         if (reference_ == "RESTRICTED") {
             dpdbuf4 T;
+            psio_->open(PSIF_OCC_DPD, PSIO_OPEN_OLD);
             global_dpd_->buf4_init(&T, PSIF_OCC_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
                                    "T2 <OO|VV>");
+            t2_diis = DIISManager(maxdiis_, "CEPA DIIS T2 Amps", DIISManager::RemovalPolicy::LargestError, DIISManager::StoragePolicy::OnDisk);
             t2_diis.set_error_vector_size(&T);
             t2_diis.set_vector_size(&T);
             global_dpd_->buf4_close(&T);
+            psio_->close(PSIF_OCC_DPD, 1);
         }
 
         else if (reference_ == "UNRESTRICTED") {
             dpdbuf4 Taa, Tbb, Tab;
+            psio_->open(PSIF_OCC_DPD, PSIO_OPEN_OLD);
             global_dpd_->buf4_init(&Taa, PSIF_OCC_DPD, 0, ID("[O,O]"), ID("[V,V]"), ID("[O,O]"), ID("[V,V]"), 0,
                                    "T2 <OO|VV>");
             global_dpd_->buf4_init(&Tbb, PSIF_OCC_DPD, 0, ID("[o,o]"), ID("[v,v]"), ID("[o,o]"), ID("[v,v]"), 0,
                                    "T2 <oo|vv>");
             global_dpd_->buf4_init(&Tab, PSIF_OCC_DPD, 0, ID("[O,o]"), ID("[V,v]"), ID("[O,o]"), ID("[V,v]"), 0,
                                    "T2 <Oo|Vv>");
+            t2_diis = DIISManager(maxdiis_, "CEPA DIIS T2 Amps", DIISManager::RemovalPolicy::LargestError, DIISManager::StoragePolicy::InCore);
```

Comment:
Is there a reason for the different the `StoragePolicy`'s for RHF and UHF?

Reply:
No... Should probably be both ``OnDisk``. I also propose that the initialization of t2_diis is taken out of the
```
if (reference_ ==  "RESTRICTED") {
...
}
else if (reference_ == "UNRESTRICTED") {
...
}
```
block. It seems redundant to me.

Reply:
I think the different policies is what Ugur had. But makes sense to unify them. I'll do it in PR 3, though, which will go up today.

Reply:
For context, during my previous changes in the `occ` module, I unified them as `behnle` suggested. This was an unintended behavior change. Lori is changing back to how Ugur had it _before_ my interventions.